core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -607,6 +608,8 @@
     public static final String WORKER_DATA_HOSTNAME = "alluxio.worker.data.hostname";
     public static final String WORKER_DATA_PORT = "alluxio.worker.data.port";
     public static final String WORKER_DATA_SERVER_CLASS = "alluxio.worker.data.server.class";
+    public static final String WORKER_DATA_SERVER_DOMAIN_SOCKET_ADDRESS =
+        "alluxio.worker.data.server.domain.socket.address";
```

Comment:
Does a worker have only a single domain socket? Are there any implications of having only 1 domain socket vs more than 1?